grift/loaders.py

```diff
@@ -143,7 +144,7 @@ def _fetch_secrets(vault_url, path, token):
         resp.raise_for_status()
         data = resp.json()
         if data.get('errors'):
-            raise VaultException(u'Error fetching Vault secrets from path {}: {}'
+            raise VaultException(six.u('Error fetching Vault secrets from path {}: {}')
```

Comment:
I don't think you need `six.u` anywhere -- unicode literals mean the same thing in both pythons